sections/03_categorize.md

```diff
@@ -167,6 +167,35 @@ advances to practical game-changers.
 Additionally, unique barriers exist in this space that may hinder progress in
 this field.
 
+###### Lack of ground-truth labels in data, and high cost in post-study validation by clinician experts.
+
+*Lack of true labels is perhaps among the biggest obstacles for electronic 
```

Comment:
I think you can remove the emphasis markers here since we were using those to mark filler text. Also check for trailing spaces (suspect you can auto-remove them - `atom`'s reflow selection will auto correct) and >80char lines. Particularly the line length can affect the ease of review.